app/views/groups/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render "shared/nav" %>
+
+<div class="transactions-container">
+  <div class="transactions-title">
+    <p class="parr"><%= @group.name %></p>
+    <p class="parr">Total: $<%= @group.expenses.sum(:amount) %></p>
+  </div>
+
+  <% @expenses.each do |expense| %>
+    <div class="transaction-card">
+      <div class="category-icon">
+        <%= image_tag @group.icon, alt: 'icon', class: "icon" %>
+      </div>
+      <div class="card-text">
+        <div class="name-amount">
+          <h4><%= expense.name %></h4>
+          <p class="amount">$<%= expense.amount %></p>
+        </div>
+        <p class="date"><%= expense.created_at.strftime('%d %B, %Y') %></p>
+      </div>
+    </div>
+  <% end %>
+
+  <%= link_to new_group_expense_path(group_id: @group.id) do %>
+    <button class="btn btn-success but">Add new transaction</button>
+  <% end %>
```

Comment:
- [ ] Same here for the Transaction page, please follow the design guidelines and make this button at the end of the page taking the full width to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/228399596-e3e8c26e-cecc-49a1-9bc6-30b4b7bf65db.png)  | ![image](https://user-images.githubusercontent.com/68796232/228399679-d6539a35-4e23-49b9-a55f-8bac253ae69f.png)  |

Reply:
Thank you @Omar-Muhamad , I'm move it to the bottom and change the size for the mobile version